recipe/meta.yaml

```diff
@@ -1,33 +1,48 @@
 {% set version = "1.0.3" %}
+{% set commit = "58fa4d15fff10f26c490518bc0da2b69732e22ee" %}
```

Comment:
I think the developer is using MPC original git repo and updates his repo from time to time with the MSVC build files rebased